projects/removingExtraWhiteSpaces/index.md

```diff
@@ -0,0 +1,46 @@
+---
+title: removingExtraWhiteSpaces in Every Language
+layout: default
+date: 2020-10-21
+last-modified: 2020-10-21
+featured-image:
+tags: [removingExtraWhiteSpaces, c]
+authors:
+    - [BHMBS Ibrahim](https://github.com/barhouum7)
+---
+
+Removing extra white spaces from a given string
+
+Given a string, remove all spaces from the string and return it with it's _length_.
+
+```c
+Input:  " Removing   Extra White   Spaces  "
+Output: "removingExtraWhiteSpaces"
+```
+
+## Requirements
+
+A string is nothing but an array of characters. The value of a string is determined by the terminating character. Its value is considered to be 0.
+
+As given in the example above, you need to enter a string first up.
+
+The string entered here is as follows: " Removing   Extra White   Spaces  "
+
+As you can see, there are quite a few blank spaces in the string entered above.
+
+Hence, the string becomes like this after removing all the blank spaces:
+
+"removingExtraWhiteSpaces"
+
+## Testing
+
+Some tests for your program are:
+
```

Comment:
Tests will be needed here at some point.

Reply:
I Added it